src/dmd/identifier.d

```diff
@@ -30,6 +30,8 @@ import dmd.utils;
 extern (C++) final class Identifier : RootObject
 {
 private:
+    __gshared Identifier anonymous_;
```

Comment:
We have `anonymous`, `anonymous_`, and `__anonymous`. Need more helpful names.

Reply:
Not sure I understand. `anonymous` is the name of the getter and I cannot have the same name of a method as an instance variable so I added the underscore, standard D naming conventions.

Reply:
This has been implicitly fixed by fixing other comments.